labeling/util.go

```diff
@@ -45,3 +45,16 @@ func loadKey(kfile string) (*did.PrivKey, error) {
 		Type: out,
 	}, nil
 }
+
+// NOTE(bnewbold): I don't know of a more idiomatic way to do this in golang
+func dedupeStrings(in []string) []string {
```

Comment:
this is fairly typical. If its performance sensitive or in a fast path there are better ways, but simple is better

Reply:
removed comment